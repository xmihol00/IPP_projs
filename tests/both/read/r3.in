a b c d 
2 8
asd
