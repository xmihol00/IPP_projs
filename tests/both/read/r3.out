a b c d 

false
